cavalier_contours/src/polyline/traits.rs

```diff
@@ -1555,9 +1557,11 @@ pub trait PlineSource {
 /// See other core polyline traits: [PlineSource] and [PlineCreation] for more information.
 pub trait PlineSourceMut: PlineSource {
     
-    fn set_userdata_values(&mut self, values: impl IntoIterator<Item = u64>);
-    
-    fn add_userdata_values(&mut self, values: impl IntoIterator<Item = u64>);
+    /// Clears the pline's userdata storage and then copies from 'values' into userdata storage.
+    fn set_userdata_values<'a>(&mut self, values: impl IntoIterator<Item = &'a u64>);
```

Comment:
API/trait signature should be `fn set_userdata_values(&mut self, values: impl IntoIterator<Item = u64>)`.

For same reasons as other signatures.